src/components/UpgradeTableOfContents.tsx

```diff
@@ -1,47 +1,10 @@
 import React from "react"
-import { motion } from "framer-motion"
-import { Link } from "gatsby"
-import styled from "@emotion/styled"
+import { Box, Link, ListItem, UnorderedList } from "@chakra-ui/react"
 
 import type { Item as ItemTableOfContents } from "./TableOfContents"
 
 const customIdRegEx = /^.+(\s*\{#([A-Za-z0-9\-_]+?)\}\s*)$/
 
-const Aside = styled.aside`
-  padding: 0rem;
-  text-align: right;
-  margin-bottom: 2rem;
-  overflow-y: auto;
-`
-
-const OuterList = styled(motion.ul)`
-  list-style-type: none;
-  list-style-image: none;
-  padding: 0;
-  margin: 0;
-  font-size: 1.25rem;
-  text-align: right;
-  line-height: 1.6;
-  font-weight: 400;
-  padding-right: 0.25rem;
-  padding-left: 1rem;
-
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    display: none;
-  }
-`
-
-const ListItem = styled.li`
-  margin: 0;
-`
-
-const StyledTableOfContentsLink = styled(Link)`
-  position: relative;
-  display: inline-block;
-  color: ${(props) => props.theme.colors.text300};
-  margin-bottom: 0.5rem !important;
```

Comment:
!important wasn't needed here, but I'm not clear how we'd implement it if it was.

Reply:
We shouldn't need to use `!important` with Chakra anymore since now, all the styles are props, and the way you override styles is by overriding those props.

Example:
```tsx
// our custom Link component
function Link(props) {
  return <ChakraLink p={4} {...props} />
}

// overriding styles
<Link p={0} />
```